backend/environment/env.content-publishing-service.template

```diff
@@ -12,8 +12,8 @@ IPFS_ENDPOINT="http://kubo_ipfs:5001"
 # IPFS_BASIC_AUTH_SECRET=
 
 # IPFS gateway URL. '[CID]' is a token that will be replaced with an actual content ID
-# IPFS_GATEWAY_URL="https://ipfs.io/ipfs/[CID]"
-IPFS_GATEWAY_URL="http://kubo_ipfs:8080/ipfs/[CID]"
+IPFS_GATEWAY_URL="https://ipfs.io/ipfs/[CID]"
+# IPFS_GATEWAY_URL="http://kubo_ipfs:8080/ipfs/[CID]"
```

Comment:
Let's just get rid of this line, since it's NEVER the right thing to have in here.
```suggestion
```